libobs/obs-hevc.h

```diff
@@ -30,6 +30,66 @@ EXPORT void obs_extract_hevc_headers(const uint8_t *packet, size_t size,
 				     uint8_t **header_data, size_t *header_size,
 				     uint8_t **sei_data, size_t *sei_size);
 
+enum {
+	OBS_HEVC_NAL_PRIORITY_DISPOSABLE = 0,
+	OBS_HEVC_NAL_PRIORITY_LOW = 1,
+	OBS_HEVC_NAL_PRIORITY_HIGH = 2,
+	OBS_HEVC_NAL_PRIORITY_HIGHEST = 3,
+};
+// NALU start codes are taken from HM 16.24 reference implementation
+// https://vcgit.hhi.fraunhofer.de/jvet/HM/-/blob/master/source/Lib/TLibCommon/TypeDef.h
+typedef enum {
```

Comment:
Do not expose enum in header if it's not public.

Reply:
Done.